content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -242,19 +242,41 @@ For Gateway deployments:
    # ...
    ```
 
-  For more information about the `passthrough` option, read [its documentation][23].
+   For more information about the `passthrough` option, read [its documentation][23].
 
 6. Make sure that the Gateway Collector's configuration uses the Datadog Exporter settings that have been removed from the agents.
```

Comment:
Can we also add an example/code-snippet here. A new customer might feel lost on this step

Reply:
Done!